lib/package/base/server-request/channels/channels_api_service.dart

```diff
@@ -243,6 +246,29 @@ class ChannelsApiService {
     return false;
   }
 
+  Future deleteChannelMessage(
+      String orgId, String channelId, String messageId, String userId) async {
+    try {
+      final res = await _api.delete(
+        '/v1/$orgId/messages/$messageId/?user_id=$userId&channel_id=$channelId',
+        token: token,
+      );
+      if (res?.statusCode == 201 || res?.statusCode == 204) {
```

Comment:
Also, confirm this status code to me too please, I think it should be 200

Reply:
![delete endpoint 2](https://user-images.githubusercontent.com/80969540/137812985-7098656d-112c-4722-a194-8b4dea585d53.png)

Though the statuscode being returned is 200 but I wrote the code based on the status code I saw on the documentation. However, kindly let me know if you need me to make any changes